crates/cibse_guide_e/src/chapter_6/appendix/a_simple_case.rs

```diff
@@ -0,0 +1,60 @@
+pub fn floor_area(w1: f64, w2: f64) -> f64 {
+    return w1 * w2
+}
+
+pub fn equation_floor_area(a_f: String, w1: String, w2: String) -> String {
+    format!("{} = {} \\cdot {}", a_f, w1, w2,)
+}
+
+pub fn area_of_opening(wo: f64, ho: f64) -> f64 {
+    return wo * ho;
+}
+
+pub fn equation_area_of_opening(a_o: String, w_o: String, h_o: String) -> String {
```

Comment:
```suggestion
pub fn area_of_opening_equation(a_o: String, w_o: String, h_o: String) -> String {
```
Same. Do what you want, this is not a blocker.